packages/remix-runtime/src/index.ts

```diff
@@ -26,12 +26,13 @@ export {
   unstable_composeUploadHandlers,
   unstable_createMemoryUploadHandler,
   unstable_parseMultipartFormData,
-  unstable_defineLoader,
-  unstable_defineAction,
   unstable_setDevServerHooks,
   UNSAFE_SingleFetchRedirectSymbol,
 } from '@remix-run/server-runtime'
 
+// TODO(serhalp) The docs say we should simply re-export all types from `/reexport`:
+// https://github.com/remix-run/remix/tree/main/packages/remix-server-runtime#readme.
+// Let's do that, but carefully verify whether this adds or removes any exports first.
```

Comment:
Shall we make a linear ticket?

Reply:
Yes. Would you mind doing that? Or we could just open a draft PR that removes them and we'll keep track of if that way.